apple/Handlers/RNPanHandler.m

```diff
@@ -31,6 +32,12 @@ @interface RNBetterPanGestureRecognizer : UIPanGestureRecognizer
 @property (nonatomic) CGFloat failOffsetYEnd;
 @property (nonatomic) CGFloat activateAfterLongPress;
 
+#if !TARGET_OS_OSX && !TARGET_OS_TV
+@property (atomic, strong) RNGHStylusData *stylusData;
+
+- (RNGHStylusData *)getStylusData;
```

Comment:
```suggestion
@property (atomic, readonly, strong) RNGHStylusData *stylusData;
```

Property should synthesize the getter

Reply:
Done in [b2a7437](https://github.com/software-mansion/react-native-gesture-handler/pull/3113/commits/b2a7437d18c51829dc29baf7cc62c65b3b419d9c)